microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -155,6 +155,30 @@ See [Keyboard shortcuts](shortcuts/index.md) and [Navigate DevTools with assisti
 You can dock DevTools in the right, left, or bottom of the browser, or undock it into a separate window.  See [Change DevTools placement (Undock, Dock to bottom, Dock to left)](customize/placement.md).
 
 
+<!-- ====================================================================== -->
+## Zooming DevTools or the rendered page
```

Comment:
@captainbrosset - There's no zoom in Settings it seems.  State that here?

Reply:
Oh you are right! I never realized that. This is a bug, I'll report it to the team. But in the meantime thanks for the docs about it.